lib/src/main/java/com/hedera/hashgraph/identity/hcs/MessageEnvelope.java

```diff
@@ -75,6 +75,10 @@ public MessageEnvelope(T message) {
             result.message = null;
         }
 
+        if (jsonNode.has(SIGNATURE_KEY) && jsonNode.get(SIGNATURE_KEY) != null) {
```

Comment:
@vijayshiyani signature was missing when message was loaded for some reason

Reply:
👍